src/test/java/org/mule/service/http/impl/service/client/GrizzlyHttpsClientProxyTestCase.java

```diff
@@ -0,0 +1,127 @@
+package org.mule.service.http.impl.service.client;
+
+import static org.mule.runtime.http.api.HttpConstants.HttpStatus.INTERNAL_SERVER_ERROR;
+import static org.mule.runtime.http.api.HttpConstants.HttpStatus.OK;
+import static org.mule.runtime.http.api.HttpConstants.Method.POST;
+import static org.mule.runtime.http.api.HttpConstants.Protocol.HTTP;
```

Comment:
import order.

Reply:
I couldn't figure out what order is wrong. I put (1) mule static imports, (2) java static imports, (3) other static imports, (4) mule imports and (5) other imports as there aren't java imports. Please, let me know where is my mistake. Thanks!

Reply:
My mistake! I was thinking of another PR when you send me this. Sorry. 😅